test_conformance/c11_atomics/test_atomics.cpp

```diff
@@ -228,11 +229,19 @@ class CBasicTestLoad : public CBasicTestMemOrderScope<HostAtomicType, HostDataTy
   }
   virtual std::string ProgramCore()
   {
-    std::string memoryOrderScope = MemoryOrderScopeStr();
-    std::string postfix(memoryOrderScope.empty() ? "" : "_explicit");
-    return
-      "  atomic_store(&destMemory[tid], tid);\n"
-      "  oldValues[tid] = atomic_load"+postfix+"(&destMemory[tid]"+memoryOrderScope+");\n";
+      // In the case this test is run with MEMORY_ORDER_ACQUIRE, the store
+      // should be MEMORY_ORDER_RELEASE
+      std::string memoryOrderScopeLoad = MemoryOrderScopeStr();
+      std::string memoryOrderScopeStore =
+          (MemoryOrder() == MEMORY_ORDER_ACQUIRE)
+          ? memoryOrderScopeLoad
+          : ", memory_order_release" + MemoryScopeStr();
```

Comment:
Are these two conditions the wrong way around, i.e. you write `X ? B : A;` but it should be `X ? A : B;` ?

Reply:
:facepalm: Updated.

Reply:
Thanks.